src/api-client.ts

```diff
@@ -20,87 +14,61 @@ const enum ContentTypes {
     SRT = 'application/x-subrip'
 }
 
-export class RevAiApiClient {
+export class RevAiApiClient extends ApiRequestHandler {
     accessToken: string;
     version: string;
-    instance: AxiosInstance;
     constructor (accessToken: string, version = 'v1') {
+        super(`https://api.rev.ai/revspeech/${version}/`, {
+            'Authorization': `Bearer ${accessToken}`,
+            'User-Agent': `RevAi-NodeSDK/2.0.1`
```

Comment:
Can we read the version from package.json now that you've changed the publish location of index.js?